cohorts/load.py

```diff
@@ -299,7 +317,7 @@ def plot_survival(self, on, col=None, col_equals=None, how="os", threshold=None)
         results = plot_kmf(
             df=df,
             condition_col=plot_col,
-            censor_col=self.dead_col if how == "os" else self.progressed_col,
+            censor_col=self.dead_col if how == "os" else self.progressed_or_dead_col,
```

Comment:
Good point. I'm curious when a separate PFS-specific censor column might result in different values than those inferred here. I believe that it could, depending on how that column got populated, but I'm fairly confused about that.